semeio/workflows/correlated_observations_scaling/job_config.py

```diff
@@ -171,48 +180,119 @@ def _CALCULATE_KEYS_key_not_empty_list(content):
-                        " that value will be used",
-                    },
-                    "alpha": {
-                        MK.Type: types.Number,
-                        MK.Description: "Scalar controlling the allowed distance"
-                        "between ensemble mean and observation. In particular, "
-                        "if: `abs(observed_value - ensemble_mean) > "
-                        "alpha * (ensenmble_std + observed_std)` the data point "
-                        "will be dropped. The value is defaulted to the value used "
-                        "in the ert run, but if a value is given in the config, "
-                        "that value will be used",
-                    },
+_CORRELATED_OBSERVATIONS_SCHEMA = {
+    MK.Type: types.NamedDict,
+    MK.Description: "Keys and index lists from all scaled keys",
+    MK.LayerTransformation: _expand_input,
+    MK.Content: {
+        "CALCULATE_KEYS": {
+            MK.Type: types.NamedDict,
+            MK.Description: _CALCULATE_KEYS_DESCRIPTION,
+            MK.Content: {
+                "keys": _KEYS_SCHEMA,
+                "threshold": {
+                    MK.Type: types.Number,
+                    MK.ElementValidators: (_min_max_value,),
+                    MK.Description: "Threshold used when computing primary"
+                    "components of the clusters.",
+                    MK.Default: 0.95,
+                },
+                "std_cutoff": {
+                    MK.Type: types.Number,
+                    MK.Description: "A lower bound on the ensemble standard"
+                    " deviation. All data points with insufficient variation"
+                    " will be dropped. The value is defaulted to the value used"
+                    " in the ert run, but if a value is given in the config, "
+                    " that value will be used",
+                },
+                "alpha": {
+                    MK.Type: types.Number,
+                    MK.Description: "Scalar controlling the allowed distance"
+                    "between ensemble mean and observation. In particular, "
+                    "if: `abs(observed_value - ensemble_mean) > "
+                    "alpha * (ensenmble_std + observed_std)` the data point "
+                    "will be dropped. The value is defaulted to the value used "
+                    "in the ert run, but if a value is given in the config, "
+                    "that value will be used",
                 },
-            },
-            "UPDATE_KEYS": {
-                MK.Type: types.NamedDict,
-                MK.Description: _UPDATE_KEYS_DESCRIPTION,
-                MK.Content: {"keys": _KEYS_SCHEMA},
             },
         },
-    }
+        "UPDATE_KEYS": {
+            MK.Type: types.NamedDict,
+            MK.Description: _UPDATE_KEYS_DESCRIPTION,
+            MK.Content: {"keys": _KEYS_SCHEMA},
+        },
+    },
+}
+
+
+class ObsCorrConfig:
```

Comment:
To be honest, the need to create a class for this instead of using the `ConfigSuite` object directly, means something is off with the original class.. We're not doing anything more than adding the validation (which could've been together with where the `SCHEMA` is located), and kind of hiding the initialisation of the configsuite object. Finally we provide just a handful of the configurations through additional functions.. 

We're at creating "yet another configuration class".. But this PR does improve the code flow, as all of this logic was just hidden within the job itself. Now we at least have the class completely without any business logic (apart from validation).

Reply:
So would you add it as some context to the `ConfigSuite` object? I guess that might be possible, but not very readable. Would you prefer I do that?

Reply:
As to getting values straight from the config object, that would be possible, the main problem is that the job might have a slight off schema, and then I am thinking of `std_cutoff` and `alfa`, that should perhaps be moved up a level, instead of being under `CALCULATE_KEYS`, but that is more of a breaking change. That would make passing configuration more natural (`config.CALULATE_KEYS` to one job and `config.FILTERING` to the other for example)

Reply:
I don't think we should add anything to the `ConfigSuite` object, I think we should keep it as is, I was just pointing out that we're kind of treating the symptoms here. No need to introduce breaking change because we're not happy with the `class` design :)